### DISCUSSION TITLE
on/off for the block

### DISCUSSION THREAD

**Discussion by alexei-moneyr** [postmanlabs/postman-flows#205] [Ideas]
Hey, could you please add on/off button to a block so if I want to skip some blocks in the flow I could do it.

![image](https://user-images.githubusercontent.com/82156498/149120970-0bb7561a-ecec-4516-90fa-e05b8d503f3f.png)

Something like this.

Thanks

**Comment by saswatds** [comment]
Thanks @alexei-moneyr for the feedback. We are working on this. @samay-v could triage this accordingly.
